opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,29 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jms</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-jms-spring</artifactId>
+      <scope>compile</scope>
+      <optional>true</optional>
```

Comment:
because it's inconvenient for users to manually specify instrumentation artifact.